<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Maps (Online)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="editorBox">
      <property name="title">
       <string>Set Marker</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblLatitude">
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Latitude">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblLongitude">
           <property name="text">
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Longitude">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblBoardName">
           <property name="text">
            <string>Board Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BoardName"/>
         </item>
         <item>
          <widget class="QLabel" name="lblSerialNumber">
           <property name="text">
            <string>Serial Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SerialNumber"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear Form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="previewButton">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="previewerBox">
      <property name="title">
       <string>Google Maps</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWebView" name="webView" native="true">
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
